R/exploratoryfactoranalysis.R

```diff
@@ -612,7 +612,7 @@ ExploratoryFactorAnalysis <- function(jaspResults, dataset, options, ...) {
       parallelResult <- try(psych::fa.parallel(dataset, plot = FALSE, fa = options$parallelMethod))
     }
     if (isTryError(parallelResult)) {
-      errmsg <- gettextf("Screeplot not available. \nInternal error message: %s", attr(pa, "condition")$message)
+      errmsg <- gettextf("Screeplot not available. \nInternal error message: %s", attr(parallelResult, "condition")$message)
```

Comment:
you can use this function from `jaspBase`
```suggestion
      errmsg <- gettextf("Screeplot not available. \nInternal error message: %s", .extractErrorMessage(parallelResult))
```